react-frontend/src/views/DoctorSignIn.js

```diff
@@ -42,16 +43,17 @@ function DoctorSignIn() {
     })
     const resData = await res.json()
     console.log(resData)
-    localStorage.setItem('doctorClinicAppToken', resData.token)
-    history.push('/home')
+    localStorage.setItem('doctorClinicAppData', JSON.stringify(resData))
+    // localStorage.setItem('doctorClinicAppPatientId', resData.token)
+    history.replace('/receptionist1')
   };
 
   return (
     <div>
-      {activityIndicator ? <LinearProgress /> : null}
       <Header />
+      {activityIndicator ? <LinearProgress style={{position: 'fixed', top: 60, width: '100%'}} /> : null}
       <div style={{display: 'flex',justifyContent:'center', margin: 5, marginTop: 100}}>
-            <div className="innerContainer" style={{backgroundColor:'#CF6A6A'}}>
+            <div className="innerContainer" style={{backgroundColor:'#CF6A6A', justifySelf:'center'}}>
```

Comment:
You can use BEM (https://css-tricks.com/bem-101/) methodology to just introduce the relevant CSS bits inside the `innerContainer ` instead of defining additional inline styles